docs/zh/SystemDesign/1-TsFile/2-Format.md

```diff
@@ -616,10 +582,14 @@ file length: 33436
 
 ### 1.4 TsFile 的总览图
 
-#### v0.8.0
+#### v0.8
 
 <img style="width:100%; max-width:800px; max-height:600px; margin-left:auto; margin-right:auto; display:block;" src="https://user-images.githubusercontent.com/33376433/65209576-2bd36000-dacb-11e9-9e43-49e0dd01274e.png">
 
-#### v0.9.0
+#### v0.9
+
+<img style="width:100%; max-width:800px; max-height:600px; margin-left:auto; margin-right:auto; display:block;" src="https://user-images.githubusercontent.com/33376433/69341240-26012300-0ca4-11ea-91a1-d516810cad44.png">
+
+#### v0.10
```

Comment:
I'm thinking if we need to change `v0.9` to `v0.9/000001`, because we write `000001` at the head of the v0.9 TsFile and `000002` the in v0.10 TsFile.
Do you think this is a good idea?

Reply:
That's a good point ! @qiaojialin What do you think?